reports/summary-2021.md

```diff
@@ -0,0 +1,1195 @@
+## Annual Report Summary 2021
+
+This is a summary of the Kubernetes project’s contributor community and
+activities. This report documents both quantitative measures of community
+health (project milestones and snapshot) as well as qualitative measures of the
+community as reported by community leaders and contributors to the project.
+
+Authors: The Kubernetes Steering Committee  
+Editor: Tim Bannister, @sftim, SIG Docs Tech Lead
+
+## Terminology
+
+This report uses the following terminology:
+
+- **Special Interest Group (SIG):** a body of contributors, responsible on an
+  ongoing basis for an area of work in the Kubernetes project. They own code,
+  docs, and/or policy.
+- **Working Group (WG):** a body of contributors, responsible for an area of work
+  in the project. Unlike SIGs, WGs dissolve once the scoped work is complete.
+  Working groups are cross-functional efforts sponsored by a SIG.
+- **Community Groups:** all of our official groups of the upstream project. Special
+  Interest Groups + Working Groups + Committees = community groups. For a full
+  list, visit the Kubernetes Contributor Site at: https://k8s.dev/groups 
+- **Chair and/or Tech Lead:** a contributor who organizes and leads a community group.
+- **Contributor:** An individual who creates an event in GitHub like a Pull Request, Issue, Review, or Comment
+- **KEP:** a [Kubernetes Enhancement Proposal][kep]
+- **OWNER:** a GitHub user who reviews, approves, and/or merges commits and is listed in an
+  [`OWNERS` file]. Maintainer is a good industry synonym.
+- **Contributor Ladder:** [member, reviewer, approver, subproject owner].
+
+
+For the community group mailing list, meeting times, and other contact info visit:
+https://k8s.dev/groups
+
+
+For community groups governance:
+  - [SIG governance]
+  - [WG governance]
+
+
+[`OWNERS` file]: https://www.kubernetes.dev/docs/guide/owners/
+[member, reviewer, approver, subproject owner]: https://git.k8s.io/community/community-membership.md
+[SIG governance]: https://git.k8s.io/community/committee-steering/governance/sig-governance.md
+[WG governance]: https://git.k8s.io/community/committee-steering/governance/wg-governance.md
+
+
+## Data collection
+
+The Kubernetes Steering Committee sent out a survey to all community group
+leads to collect data for this report. Each individual group report may be
+found in their respective directory inside the [Kubernetes Community repo].
+
+For more, see:
+[Program Documentation]
+
+[Program Documentation]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/annual-reports.md
+
+## Contributor snapshot
```

Comment:
not sure the newlines render in markdown as you want when there's multiple items in a paragraph ("62000 contributors all time 10000 new contributors this year")